ci-operator/config/stackrox/kernel-packer/stackrox-kernel-packer-ddolgov-feature-crawling-on-vm.yaml

```diff
@@ -0,0 +1,68 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: kernel-crawling
+  steps:
+    post:
+    - as: delete-vm
+      commands: .openshift-ci/crawler/post.sh
+      credentials:
+      - mount_path: /tmp/secret/stackrox-kernel-packer-crawl
+        name: stackrox-kernel-packer-crawl-test
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 4000m
+          memory: 4000Mi
```

Comment:
I think you can probably modify the global `resources` requested and remove the ones for individual steps, its a minor thing but I think it would help reduce some clutter.

Reply:
Just have tried that, make update says every step has to specify at least one request or limit.